doc/source/operating/hints.rst

```diff
@@ -1,22 +1,184 @@
+|                      |has failed.                                |                 |                                                   
++----------------------+-------------------------------------------+-----------------+
+|hinted_handoff        |Maximum throttle in KBs per second, per    |                 |
+|_throttle_in_kb       |delivery thread. This will be reduced      | 1024            |
+|                      |proportionally to the number of nodes in   |                 | 
+|                      |the cluster.                               |                 |
+|                      |(If there are two nodes in the cluster,    |                 |
+|                      |each delivery thread will use the maximum  |                 |
+|                      |rate; if there are 3, each will throttle   |                 |
+|                      |to half of the maximum,since it is expected|                 |
+|                      |for two nodes to be delivering hints       |                 |
+|                      |simultaneously.)                           |                 |
++----------------------+-------------------------------------------+-----------------+
+|max_hints_delivery    |Number of threads with which to deliver    |     2           |
+|_threads              |hints; Consider increasing this number when|                 |
+|                      |you have multi-dc deployments, since       |                 |
+|                      |cross-dc handoff tends to be slower        |                 |
++----------------------+-------------------------------------------+-----------------+
+|hints_directory       |Directory where Cassandra stores hints.    |$CASSANDRA_HOME/ |
+|                      |                                           |data/hints       |
++----------------------+-------------------------------------------+-----------------+
+|hints_flush_period_in |How often hints should be flushed from the |  10000          |
+|_ms                   |internal buffers to disk. Will *not*       |                 |
+|                      |trigger fsync.                             |                 |
++----------------------+-------------------------------------------+-----------------+
+|max_hints_file_size   |Maximum size for a single hints file, in   |   128           |
+|_in_mb                |megabytes.                                 |                 |
++----------------------+-------------------------------------------+-----------------+
+|hints_compression     |Compression to apply to the hint files.    |  LZ4Compress    | 
+|                      |If omitted, hints files will be written    |                 |
+|                      |uncompressed. LZ4, Snappy, and Deflate     |                 |
+|                      |compressors are supported.                 |                 |
++----------------------+-------------------------------------------+-----------------+
+ 
+Changing Max Hint Window at Runtime

```

Comment:
Can we change this section to talk about when you may want more time for hints to play instead of changing max hint window at runtime? It's actually somewhat rare for nodes to be down for more than three hours but its very common for hints playing at 1024 `kbps` cannot complete before 3 hours.

You could mention raising the `hinted_handoff_throttle` as well as raising the window to ensure hints are delivered.

Reply:
modified.